com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -0,0 +1,96 @@
+| **Detect Paper White**               | Whether the paper white value is detected from the data communicated by the display device. We strongly suggest to provide a calibration screen to let the user set this value depending on their screen and viewing experience. |
+| **- Paper White**                    | Paper white value (in nits) set when it is not automatically detected from the display. |
+| **Detect Brightness Limits**         | Whether the minimum and maximum brightness values are detected from the data communicated by the display device. While using these values as detected lead to more precise results than detecting paperwhite values, we still suggest to use a calibration screen. |
+| **- Min Nits**                       | Minimum brightness supported by the device.                  |
+| **- Max Nits**                       | Maximum brightness supported by the device.                  |
+
+While Hue-preserving tonemapping (i.e. Hue Shift Amount set to 0) will better preserve the content colors, sometimes content is authored to rely on the hue shifts that high brightness will produce. A typical example can be for example a very bright flame VFX. The image below shows on the left a flame with Hue Shift Amount set to 0 and on the right the same flame with the hue shift preserved (Hue Shift Amount set to 1).
+
+ <img src="C:\Users\franc\Documents\Github\SRP\com.unity.render-pipelines.high-definition\Documentation~\Images\HDR-Output-HueShift.png" alt="HDR-Output-HueShift" style="zoom:67%;" />
+*Image modified to show the issue clearly.*
+
+The right choice depends on your content, but we suggest to make sure your content is authored to work well without relying on Hue-shift.
+
+#### ACES
+
+Contrary to the neutral version, the ACES tonemapping option has fixed presets. These are meant to target screens with 1000, 2000 and 4000 nits maximum brightness. We suggest to use calibration screen to detect the right preset for the user consuming the content.
+
+![HDR-Output-ACES](C:\Users\franc\Documents\Github\SRP\com.unity.render-pipelines.high-definition\Documentation~\Images\HDR-Output-ACES.png)
+
+##
+
+| **Property**           | **Description**                                              |
+| ---------------------- | ------------------------------------------------------------ |
+| **ACES Preset**        | The tonemapper preset to use. The options are:<br />- ACES 1000 Nits: Curve used to target 1000 nits displays (Default)<br />- ACES 2000 Nits: Curve used to target 2000 nits displays <br />- ACES 4000 Nits: Curve used to target 4000 nits displays |
```

Comment:
small adjustments:
"ACES 1000 Nits: The default. Curve for 1000 nits displays."
instead of "used to target" use "for", then we can avoid passive voice (: